extensions/resteasy-reactive/quarkus-resteasy-reactive-jsonb/runtime/src/main/java/io/quarkus/resteasy/reactive/jsonb/runtime/serialisers/JsonbMessageBodyReader.java

```diff
@@ -53,4 +53,15 @@ private Object doReadFrom(Class<Object> type, Type genericType, InputStream enti
         }
         return json.fromJson(entityStream, genericType != null ? genericType : type);
     }
+
+    private boolean isReadable(MediaType mediaType, Class<?> type, Type genericType) {
```

Comment:
Sounds like this could be abstracted somewhere common. Especially given that it has no json dependency, we could put this in `-common`

Reply:
Will do